File: web/src/app/poto/place.ts
/**
 * 瑜伽场馆
 */
export class Place {
    constructor(
        public id?: number,
        public phone?: string,
        public sName?: string,
        public address?: string,
        public picUrl?: string,
        public intro?: string
    ) {}

    static fromJSON(json: any): Place {
        const place = Object.create(Place.prototype);
        Object.assign(place, json);
        return place;
    }
    
}

File: app/app/src/main/java/bean/Place.java
package bean;

/**
 * 场馆地址
 * Created by Thor on 2018/4/4.
 */

public class Place {

    private Integer id;

    private String phone;

    private String sName;

    private String address;

    private String picUrl;

    private String intro;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone == null ? null : phone.trim();
    }

    public String getsName() {
        return sName;
    }

    public void setsName(String sName) {
        this.sName = sName == null ? null : sName.trim();
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address == null ? null : address.trim();
    }

    public String getPicUrl() {
        return picUrl;
    }

    public void setPicUrl(String picUrl) {
        this.picUrl = picUrl;
    }

    public String getIntro() {
        return intro;
    }

    public void setIntro(String intro) {
        this.intro = intro;
    }
}